src/Magnum/OvrIntegration/HmdEnum.h

```diff
@@ -224,30 +424,55 @@ enum class ErrorType: Int {
-    BootloaderDeviceDetected = ovrError_BootloaderDeviceDetected,   /**< A bootloader HMD is detected by the service */
-    TrackerCalibrationError = ovrError_TrackerCalibrationError,     /**< The tracker calibration is missing or incorrect */
-    ControllerFirmwareMismatch = ovrError_ControllerFirmwareMismatch,/**< The controller firmware is out of date and is unacceptable */
+    BootloaderDeviceDetected = ovrError_BootloaderDeviceDetected,   /**< A bootloader HMD is detected by the service. */
+    TrackerCalibrationError = ovrError_TrackerCalibrationError,     /**< The tracker calibration is missing or incorrect. */
+    ControllerFirmwareMismatch = ovrError_ControllerFirmwareMismatch,/**< The controller firmware is out of date and is unacceptable. */
+
+    IMUTooManyLostSamples = ovrError_IMUTooManyLostSamples,         /**< Too many lost IMU samples. */
+    IMURateError = ovrError_IMURateError,                           /**< IMU rate is outside of the expected range. */
+    FeatureReportFailure = ovrError_FeatureReportFailure,           /**< A feature report has failed. */
 
     /* Synchronization Errors */
     Incomplete = ovrError_Incomplete,   /**< Requested async work not yet complete. */
     Abandoned = ovrError_Abandoned,     /**< Requested async work was abandoned and result is incomplete. */
 
     /* Rendering Errors */
-    DisplayLost = ovrError_DisplayLost, /**< In the event of a system-wide graphics reset or cable unplug this is returned to the app */
+    DisplayLost = ovrError_DisplayLost,                             /**< In the event of a system-wide graphics reset or cable unplug this is returned to the app. */
+    TextureSwapChainFull = ovrError_TextureSwapChainFull,           /**< ovr_CommitTextureSwapChain was called too many times on a texture swapchain without calling submit to use the chain. */
+    TextureSwapChainInvalid = ovrError_TextureSwapChainInvalid,     /**< The ovrTextureSwapChain is in an incomplete or inconsistent state. Ensure @ref TextureSwapChain::commit() was called at least once first. */
 
     /* Fatal errors */
     /**
      * A runtime exception occurred. The application is required to shutdown
      * LibOVR and re-initialize it before this error state will be cleared.
      */
     RuntimeException = ovrError_RuntimeException,
+
+    MetricsUnknownApp = ovrError_MetricsUnknownApp,
+    MetricsDuplicateApp = ovrError_MetricsDuplicateApp,
+    MetricsNoEvents = ovrError_MetricsNoEvents,
+    MetricsRuntime = ovrError_MetricsRuntime,
+    MetricsFile  = ovrError_MetricsFile,
+    MetricsNoClientInfo = ovrError_MetricsNoClientInfo,
+    MetricsNoAppMetaData = ovrError_MetricsNoAppMetaData,
+    MetricsNoApp = ovrError_MetricsNoApp,
+    MetricsOafFailure = ovrError_MetricsOafFailure,
+    MetricsSessionAlreadyActive = ovrError_MetricsSessionAlreadyActive,
+    MetricsSessionNotActive = ovrError_MetricsSessionNotActive,
```

Comment:
I just found lwjgl also wraps the Oculus SDK and they document these as `Error codes (ovrErrorType)`...

Some of these I can't even imagine what they are supposed to mean.